spec/rest/http.test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+define(['ably', 'shared_helper'], function(Ably, helper) {
+	var rest, exports = {},
+		Defaults = Ably.Rest.Defaults;
+
+	exports.setupHttp = function(test) {
+		test.expect(1);
+		helper.setupApp(function() {
+			rest = helper.AblyRest();
+			test.ok(true, 'App created');
+			test.done();
+		});
+	}
+
+	/**
+	 * Check presence of X-Ably-Version headers in get&post requests
+	 * @spec : (RSC7a)
+	 */
+	exports.apiVersionHeader = function(test) {
+
+		//Intercept get&post methods with test
+		Ably.Rest.Http.get_inner = Ably.Rest.Http.get;
```

Comment:
Not sure why the original needs to be stored as a new attr on the Http object, why not just in a local variable?